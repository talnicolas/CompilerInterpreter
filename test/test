int fact(int n) {
  if(n == 0) {
    return 1;
  }
  return n * fact(n - 1);
}

{
  int i = 3;
  print(i);
  println();
  if(false) {
    int j = 2;
  } else {
    i = fact(5);
  }
  print("le resultat est " + string(i));
  println();

  i = 0;
  while(i < size(args)) {
    print(args[i]);
    println();
    i = i + 1;
  }
}


