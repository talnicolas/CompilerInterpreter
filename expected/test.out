3
le resultat est 3
